solr/core/src/java/org/apache/solr/cli/ExportTool.java

```diff
@@ -202,7 +209,16 @@ DocsSink getSink() {
     abstract void exportDocs() throws Exception;
 
     void fetchUniqueKey() throws SolrServerException, IOException {
-      solrClient = new CloudHttp2SolrClient.Builder(Collections.singletonList(baseurl)).build();
+      Http2SolrClient.Builder builder =
+          new Http2SolrClient.Builder()
+              .withIdleTimeout(30, TimeUnit.SECONDS)
```

Comment:
while it seems reasonable for Create and Delete to have the timeout values you used here, I am wondering if the Export is not too short? don't really have a good idea, just raising this for evaluation again.

Reply:
Good question.  I removed them since there are some defaults defined.....